app/javascript/toolbar-actions/custom-action.js

```diff
@@ -15,6 +15,15 @@ export function APIFunction(actionType, entity, resources) {
       } else {
         add_flash(sprintf(__('Requested %s of selected item.'), actionType), 'success');
       }
+
+      if (data.results){
+        data.results.forEach(res => {
+          if (!res.success) {
+            add_flash(sprintf(__(res.message)), 'error');
+          }
+        });
+      }
```

Comment:
I think this would be unnecessary after @kbrock fixes https://github.com/ManageIQ/manageiq-api/issues/919

Reply:
Actions such as delete are queued so the only msg you get is "Requested delete of selected item" but in some cases the physical storage does not support the action and the user wont see that it fails. for example, if we have a lot of physical storages from different kind and we label them for deletion but not all supports delete, we will get one msg   "Requested delete of selected item", but I think that we should also get additional msg that says if a specific storage failed. so this short fix  just show the error like below : 
 
<img width="644" alt="1" src="https://user-images.githubusercontent.com/53213107/98647310-5884af80-233d-11eb-817d-d23333cc607b.png">

If https://github.com/ManageIQ/manageiq-api/issues/919 is similar, or if you disagree please let me know and I will make the required changes

Reply:
Let's wait for 919 to get in, the goal there is to respond with a non-200 HTTP response which you can't handle in a `.then` but you need a `.catch`. But we might have some kind of implicit handling on it...hard to tell without this getting in.

Reply:
ok, so I will remove this change from this PR, and wait to see what will be with PR-919.

Reply:
OR we get this in and we will get a surprise when 919 gets in, your call, depends on how fast you need this...

Reply:
I think that the changes in custom-action.js are not related to the issue of removing physical storage so it can be in different PR. we just wont know if the deletion fails (but we dont know about any failed queued action). maybe you can add comment in PR-919 about this discussion ?

anyway I think I will remove it from here